css-shapes-2/Overview.bs

```diff
@@ -165,6 +168,79 @@ Supported Shapes</h3>
 					in the path string.
 					For the initial direction follow SVG 1.1.
 			</ul>
+		<dt>
+			<pre class=prod>
+				<dfn>shape()</dfn> = shape( [<<fill-rule>>,]? from <<coordinate-pair>>, <<draw-command>>#)
+			</pre>
+		<dd dfn-type=value dfn-for="shape()">
+			<ul>
+				<li>
+					The <<coordinate-pair>> represents the starting point for the first draw-command,
+					in physical coordinates.
+				<li>
+					The sequence of <dfn><<draw-command>></dfn>s represent commands of an
+					<a href="https://www.w3.org/TR/SVG11/paths.html#PathData">SVG Path</a>.
+			</ul>
+	</dl>
+	
+	The arguments not defined above are defined as follows:
+
+	<dl>
+		<dt><dfn><<coordinate-pair>></dfn> = <<length-percentage>>{2}
+		<dd>Defines a pair of coordinates x & y.
```

Comment:
Should this be a `<<position>>`?

Reply:
Sure

Reply:
Actually, the problem is that <<position>> contains `left`/`top` etc, which don't make sense for relative coordinates. WDYT?

Reply:
Hmm, to handle this properly we'd need to split out the abs/rel at a higher level, rather than just via the `by`/`to` keyword, so we could use the right non-terminal in each branch. Should be a fairly light bit of editting, just would make the spec a bit more verbose.

Reply:
Got it, I'll revise.

Reply:
Ah, agreed. Definitely shouldn't have `<<position>>` everywhere.

@tabatkins Not sure what you mean, do you mean like the `<<corner>>` proposal earlier? I think that made a lot of sense, actually, switching the entire path into a different coordinate system so both absolute and relative coordinates switch over. Though doing that in a separate issue/PR also makes sense.

Reply:
New revision tackles this.

Reply:
@tabatkins @fantasai I'm not sure which one I prefer, using `<<position>>` or limiting to coordinate pairs.

On one hand, `<<position>>` allows us to use ``left``, ``center`` etc. OTOH, when we add logical-property support to shapes, e.g. as the previous `<<corner>>` proposal, they would make less sense. Also `polygon()` uses `<<length-percentage>>` and not `<<position>>`, so maybe for the sake of consistency it's best to leave shapes with `<<coordinate-pair>>` for the time being. WDYT?

Reply:
Yes, agree to leave as <<coordinate-pair>> here. We can always expand it out later if we decide it's a good idea, but I think it's good to evaluate that separately (and in consideration of what we want to do for logical coords).

Reply:
Ok. Reverted `<<position>>` change. I think the patch is ready for being on the agenda!